### PR TITLE
Task/AKDS-2/accessibility

### PR DESCRIPTION
**Pull request by ksanmartin909**
You may notice in the CloseUp satellite detail, the static satellite images are missing.
This was a side effect of the breakpoint at 1170 and is fixed in PR #20 

opera_MhXLJRKLbb

opera_5gM51PNFtq

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -149,8 +149,8 @@ export const App: React.FC = () => {
           <div className="nav_container">
             {!flyToId && (
               <div className="orbit_title_container">
-                {orbitIds.map((id, index) => {
-                  return <OrbitTitleGroup key={id} id={id} tabIndex={index} />;
+                {orbitIds.map((id) => {
+                  return <OrbitTitleGroup key={id} id={id} tabIndex={0} />;
```
Comment (ksanmartin909) [inline src/App.tsx:153]: Apparently  adding a `tabIndex` value of higher than 0 just forces the browser to tab to the element out of the order it appears in the html document and should be used very sparingly.  

`tabIndex: 0` makes sure it should be accessible in the order it appears on the html page.
`tabIndex: -1` means the element should not be accessible with the tab key.

File: src/components/OrbitCloseupText.tsx [hunk 2]
```diff
@@ -159,24 +165,41 @@ const OrbitCloseupText: React.FC = () => {
...
       </div>
       <img
         className="orbit_closeup_back_button"
         src={close}
         onClick={backtoMain}
+        alt="close button"
+        tabIndex={0}
+        onKeyDown={(e) => {
+          e.key === "Enter" && backtoMain();
+        }}
```
Comment (ksanmartin909) [inline src/components/OrbitCloseupText.tsx:189]: Allows the Enter key to trigger a mouseClick.

File: src/components/PopUpText.tsx [hunk 3]
```diff
@@ -46,7 +46,7 @@ const PopUpText: React.FC = () => {
     <div className="orbit_title_popup_container">
       <div className="orbit_title_popup">
         <div className="orbit_title_info_header">{headerText}</div>
-        <div className="orbit_title_info_body">
+        <div className="orbit_title_info_body" tabIndex={0}>
```
Comment (ksanmartin909) [inline src/components/PopUpText.tsx:49]: It was telling me that txt or body elements that require scrolling need to be keyboard accessible.

File: src/components/RiskIcons.tsx [hunk 4]
```diff
@@ -1,62 +1,81 @@
...
+  const renderedImages = images.map((image, index) => (
+    <a
+      key={imageDescs[index]}
+      data-tooltip-id={imageDescs[index]}
+      data-tooltip-content={formatTooltipContent(imageDescs[index])}
+    >
+      <img
+        src={image}
+        className={"risk_icon"}
+        alt={imageDescs[index] + "_risk_icon"}
```
Comment (ksanmartin909) [inline src/components/RiskIcons.tsx:72]: A bunch of reformatting in this component just to add an alt to the risk_icon

**Review by rraaschfilament — APPROVED** [review]
Looks good!

**Pull request merged**